.travis.yml

```diff
@@ -21,6 +21,7 @@ cache:
     - $HOME/.composer/cache
 
 before_install:
+  - if [[ "$WITH_COVERAGE" != "true" ]]; then phpenv config-rm xdebug.ini; fi
```

Comment:
@mdwheele 

Depending on whether you like #298, this condition should be adjusted, of course!